dapp/pages/_app.js

```diff
@@ -103,9 +103,10 @@ function App({ Component, pageProps, err }) {
     }
   }, [])
 
-	useEffect(() => {
-		if (localStorage.locale) {
-			setLocale(localStorage.locale)
+  useEffect(() => {
+    if (localStorage.locale) {
+      setLocale(localStorage.locale)
+      setUtilLocale(localStorage.locale)
```

Comment:
The `setUtilLocale` code triggers a `analytics.track` event, since it doesn't predict that it can be called on initialise. Can you add a boolean param to the utils `setLocale` that will distinguish between initialise call and user initiated call. And not track the analytics event when it has been called during initialisation?

Reply:
@sparrowDom 
Thank you! I have just updated it.
Can you please check the latest update?

Reply:
Look good thanks for fixing it